recipes-kernel/kernel-tests/kernel-performance-tests-files/common.cfg

```diff
@@ -0,0 +1,6 @@
+# Common configuration for all kernel performance tests
+TEST_DURATION=8h
```

Comment:
I suppose this isn't critical for what we're intending to use this ptest suite for, but it's worth noting that the `ptest-runner` wrapper binary is currently hardcoded to ETIMEOUT a test suite [if it runs for more than 300 seconds](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/3560/Yocto-PTests?anchor=timeliness). So we'll need to invoke this test manually in any testsuite we use it in (and/or upstream a patch to `ptest-runner` to allow for modifying that timeout).

Reply:
ptest-runner already has that option ('-t'). I'm currently currently doing a run using that option i.e. 'ptest-runner -t 360000' which should give me enough runway for these tests before it times out.

Reply:
Ah! Nevermind; I have old information then.